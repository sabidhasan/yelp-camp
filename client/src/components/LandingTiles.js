import React from 'react'

const LandingTiles = () => {
  return (
    <div className='landing-text landing-tile'>
      <section>
        <i>Icon</i>
        <h2>1550</h2>
        <p>Campgrounds</p>
      </section>

      <section>
        <i>Icon</i>
        <h2>1</h2>
        <p>Easy-to-use Source</p>
      </section>

      <section>
        <i>Icon</i>
        <h2>13</h2>
        <p>Provinces and Territories</p>
      </section>

      <section>
        <i>Icon</i>
        <h2>1.09</h2>
        <p>million Canadians camp</p>
      </section>
    </div>
  )
}

export default LandingTiles
