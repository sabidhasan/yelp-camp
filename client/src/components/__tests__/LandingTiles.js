import React from 'react'
import toJson from 'enzyme-to-json'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import LandingTiles from '../LandingTiles'

configure({ adapter: new Adapter() });

describe('<LandingTiles />', () => {
  const component = shallow(<LandingTiles />)

  it('Component should be defined', () => {
    expect(component).toBeDefined();
  });

  it('LandingTiles component snapshot matches what is expected', () => {
    expect(toJson(component)).toMatchSnapshot();
  })
})
