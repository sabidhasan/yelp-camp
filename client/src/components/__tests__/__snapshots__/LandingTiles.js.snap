// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingTiles /> LandingTiles component snapshot matches what is expected 1`] = `
<div
  className="LandingTiles LandingText"
>
  <section
    className="LandingTiles__section LandingText__section"
  >
    <i
      className="LandingTiles__icon fas fa-home"
    />
    <h2
      className="LandingTiles__number bold"
    >
      1550
    </h2>
    <p
      className="LandingTiles__text"
    >
      Unique Campgrounds
    </p>
  </section>
  <section
    className="LandingTiles__section LandingText__section"
  >
    <i
      className="LandingTiles__icon far fa-bookmark"
    />
    <h2
      className="LandingTiles__number bold"
    >
      1
    </h2>
    <p
      className="LandingTiles__text"
    >
      Easy-to-Use Source
    </p>
  </section>
  <section
    className="LandingTiles__section LandingText__section"
  >
    <i
      className="LandingTiles__icon far fa-map"
    />
    <h2
      className="LandingTiles__number bold"
    >
      12
    </h2>
    <p
      className="LandingTiles__text"
    >
      Provinces and Territories
    </p>
  </section>
</div>
`;
